admin-dev/themes/default/js/admin-theme.js

```diff
@@ -454,25 +454,6 @@ $(document).ready(() => {
     $('#bo_query').focus();
   });
 
-  // reset form
-  /* global header_confirm_reset, body_confirm_reset, left_button_confirm_reset, right_button_confirm_reset */
```

Comment:
This is related to the module reset button that is commented out since least since 1.6 (see https://github.com/PrestaShop/PrestaShop/blob/1.7.8.0/admin-dev/themes/default/template/controllers/modules/configure.tpl)